examples/examples_overview/examples_overview/app.py

```diff
@@ -106,7 +106,8 @@ def startup(self):
             ),
         )
 
-        split_container = toga.SplitContainer(content=[left_box, self.info_view])
+        # make the SplitContainer flex to occupy available Box widget space
+        split_container = toga.SplitContainer(style=Pack(flex=1), content=[left_box, self.info_view])
```

Comment:
The structure of the app is as following:
outer_box
|...Label
|...SpliterContainer
|...|...left_box (Box widget)
|...|...|...table (Table widget)
|...|...|...button_box (Box widget has 2 button)
|...|...info_view (MultilineTextInput widget)

Are we need to tell the `SpliterContainer` to occupy available  space of `outer_box`? Why it will occupy available space without specify `flex=1`.

Reply:
It's not as straightforward as that. `outer_box` is also the main content of the window, and the layout must conform to the height of the window. The only content in the box is a label and a splitter container, and of those two, label has a fixed height. As a result, the splitter will must absorb all the remaining height. `flex=1` should have no effect here - there no ambiguity over where the expansion weight can go.

By way of (weak) proof - macOS and Windows both render this layout the same, and adding flex=1 changes nothing.

There is perhaps a design argument to be made that the interaction between outer_box and window size, and whether it should be possible to specify a window axis size of "None" that causes the window to conform to the minimum possible size; but that's definitely in territory of "feature", rather than "bug".